src/module_ncio.f90

```diff
@@ -5,52 +5,77 @@
 !! variables and char variables. Variables can have up to 5 dimensions.
 !! @author jeff whitaker <jeffrey.s.whitaker@noaa.gov> @date 201910
 module module_ncio
-
+!
   use netcdf
   use mpi
-
+!
   implicit none
   private
-
+!
```

Comment:
Please don't add regular comments or change the spacing of the code.

Reply:
addressed